.circleci/config.yml

```diff
@@ -298,91 +319,66 @@ jobs:
 
 workflows:
   version: 2
-  build_and_test:
+
+  Deploy_to_Development:
```

Comment:
Is it possible to apply the filter to the whole workflow?

Reply:
Not as a key under `workflows/<workflow_name>` at the moment, [no](https://circleci.com/docs/configuration-reference/#workflows).